graphql/school/school.model.js

```
@@ -0,0 +1,68 @@
+// *************** IMPORT LIBRARY ***************
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const SchoolSchema = new Schema({
+  // Unique identifier for the school
+  _id: {
+    type: Schema.Types.ObjectId,
+    auto: true
+  },
+
+  // Short Name of the school
+  short_name: {
+    type: String,
+    required: true,
+    unique: true,
+    trim: true,
+    default: '',

```

default: '' on required: true fields is misleading